scipy/stats/morestats.py

```diff
@@ -182,23 +182,21 @@ def kstat(data, n=2):
     See Also
     --------
     kstatvar: Returns an unbiased estimator of the variance of the k-statistic.
+    moment: Returns the n-th central moment about the mean for a sample.
 
     Notes
     -----
-    The cumulants are related to central moments but are specifically defined
-    using a power series expansion of the logarithm of the characteristic
-    function (which is the Fourier transform of the PDF).
-    In particular let phi(t) be the characteristic function, then::
+    For a sample size n, the first few k-statistics are given by:
 
-        ln phi(t) = > kappa_n (it)^n / n!    (sum from n=0 to inf)
+    .. math::
 
-    The first few cumulants (kappa_n)  in terms of central moments (mu_n) are::
+        k_{1} = \mu
+        k_{2} = \frac{n}{n-1} m_{2}
+        k_{3} = \frac{ n^{2} } {(n-1) (n-2)} m_{3}
+        k_{4} = \frac{ n^{2} [(n + 1)m_{4} - 3(n - 1) m^2_{2}]} {(n-1) (n-2) (n-3)}
 
-        kappa_1 = mu_1
-        kappa_2 = mu_2
-        kappa_3 = mu_3
-        kappa_4 = mu_4 - 3*mu_2**2
-        kappa_5 = mu_5 - 10*mu_2 * mu_3
+    where ``\mu`` is the sample mean, ``m_{2}`` is the sample variance, and
+    ``m_{i}`` is the i-th sample central moment.
```

Comment:
This mixes the code markup (double back ticks) and TeX, so better do :math:`\m_2` and so on.